scripts/spark_eval/copy_small_to_hdfs_on_gcs.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# Copy small data to HDFS on GCS
+
+${GATK_HOME:-../..}/gatk-launch ParallelCopyGCSDirectoryIntoHDFSSpark \
```

Comment:
Neat, I didn't know about this bashism.   Alternatively you can add gatk-launch to your path.